recipes/spirv-cross/config.yml

```diff
@@ -7,29 +13,5 @@ versions:
     folder: all
   "1.3.231.1":
     folder: all
-  "1.3.224.0":
```

Comment:
Please keep this version as it is a requirement for "diligent-core" recipe https://github.com/conan-io/conan-center-index/blob/91e67bdec23d59d4e2a104a3343762142b300058/recipes/diligent-core/all/conanfile.py#L116